src/AccessibilityInsights.SharedUx/Controls/InspectTabsControl.xaml

```diff
@@ -76,7 +76,8 @@
                                     <ColumnDefinition Width="*"/>
                                     <ColumnDefinition Width="Auto"/>
                                 </Grid.ColumnDefinitions>
-                                <Label Grid.Column="0" Margin="4" Style="{StaticResource ResourceKey=lblAccordionHeader}" Target="{Binding RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type ToggleButton}}}" Content="{x:Static Properties:Resources.ExpanderAutomationPropertiesNameProps}" />
+                                <Label Grid.Column="0" Margin="4" Style="{StaticResource ResourceKey=lblAccordionHeader}"
```

Comment:
Interesting. According to https://learn.microsoft.com/en-us/dotnet/api/system.windows.controls.label.target, The Target property specifies the element that receives the focus when the accelerator key is pressed. This Label has no accelerator key, so it's a NOP either way. Removing it makes sense to me.